tests/really-object-spec.coffee

```diff
@@ -0,0 +1,194 @@
+# Dependencies
+
+Q            = require 'q'
+ReallyObject = require '../src/really-object'
+ReallyError  = require '../src/really-error'
+protocol     = require '../src/protocol'
+
+describe 'really-object', ->
+
+  channel =
+    send: -> true
+
+  describe 'get', ->
+
+    it 'should throw error if called without passing resource parameter', ->
+      expect ->
+        object = new ReallyObject()
+        msg = object.get()
+      .toThrow new ReallyError('Can not be initialized without resource')
+
+    it 'should throw error if called with non-string resource parameter', ->
+      expect ->
+        object = new ReallyObject()
+        msg    = object.get(123)
```

Comment:
it'd be better if you just put what you expect to throw error in expect not the whole code; for clarity
```coffeescript
object = new ReallyObject()
expect ->
  object.get(123)
...
```